test/powershell/Host/PSVersionTable.Tests.ps1

```diff
@@ -1,6 +1,6 @@
 Describe "PSVersionTable" -Tags "CI" {
     It "Should have version table entries" {
-	$PSVersionTable.Count | Should Be 9
+	$PSVersionTable.Count | Should Be 11
```

Comment:
Please fix the indentation within this file. It looks like you may have added tab characters instead of spaces. My comment applies to all changes made within this file.